### PR TITLE
remove azure jobs

### PR DESCRIPTION
**Pull request by jim-minter**
removing for now because generally broken and additionally not cleaning up properly - will revert this PR later to re-enable

### CODE REVIEW

**Comment by jim-minter** [comment]
@smarterclayton ptal

File: cluster/ci/config/prow/config.yaml [hunk 1]
```diff
@@ -393,32 +393,6 @@ presubmits:
     rerun_command: "/test service-catalog"
     trigger: "((?m)^/test service-catalog,?(\\s+|$))"
 
-  - name: test_pull_request_origin_extended_conformance_azure
-    agent: jenkins
-    labels:
-      master: ci.openshift.redhat.com
-    always_run: true
```
Comment (kargakis) [inline cluster/ci/config/prow/config.yaml]: You can just switch this to false instead of removing them altogether.

Comment (jim-minter) [reply cluster/ci/config/prow/config.yaml]: I didn't know what the equivalent of that was for the postsubmits and just wanted to get them disabled - does it matter?

Comment (kargakis) [reply cluster/ci/config/prow/config.yaml]: ok, fine by me

**Pull request merged**